server/src/test/java/io/crate/analyze/SelectStatementAnalyzerTest.java

```diff
@@ -1920,7 +1920,7 @@ public void testExtractFunctionWithWrongType() throws Exception {
         AnalyzedRelation relation = executor.analyze(
             "select extract(day from name::timestamp with time zone) from users");
         Symbol symbol = relation.outputs().get(0);
-        assertThat(symbol).isFunction("extract_DAY_OF_MONTH");
+        assertThat(symbol).isFunction("extract_DAY");
```

Comment:
not a bug, for timestamps `DAY` == `DAY_OF_MONTH`, but since `DAY_OF_MONTH` doesn't make sense for intervals, only `DAY` I had to split them, but for timestamps they're still the same function.